srcpkgs/libbitcoin-consensus/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'libbitcoin-consensus'
+pkgname=libbitcoin-consensus
+version=3.5.0
+revision=1
+build_style=gnu-configure
+configure_args="ax_cv_boost_unit_test_framework=no --with-gnu-ld"
+hostmakedepends="autoconf automake libtool m4 pkg-config"
+makedepends="boost-devel libsecp256k1-devel"
+short_desc="Bitcoin Consensus Library"
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+license="AGPL-3.0-or-later"
+homepage="https://github.com/libbitcoin/libbitcoin-consensus"
+distfiles="https://github.com/libbitcoin/$pkgname/archive/v$version.tar.gz>$pkgname-$version.tar.gz"
+checksum=bb29761d4275a9c993151707557008b23572a3d9adecc0e36a3075cfb101dd1e
+
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" --host=$XBPS_CROSS_TRIPLET --with-boost=$XBPS_CROSS_BASE/usr
+ --with-boost-libdir=$XBPS_CROSS_BASE/usr/lib"
```

Comment:
These can be in the normal configure_args. it will expand to nothing in native builds